qml/utils.js

```diff
@@ -8,3 +8,41 @@ function versionGE(version, major, minor, micro) {
   }
   return false
 }
```

Comment:
@dagheyman Should I change to 4-space indentation here too?

Reply:
I just ran autoformat on it, it did some things.